### PR TITLE
Add VT100 driver for Atari.

### PR DESCRIPTION
**Pull request by groessler**
A special font will be needed for best user experience. This is not
included yet, but will come soon...

### CODE REVIEW

File: drivers/atrvt100.s [hunk 1]
```diff
@@ -1,6 +1,1709 @@
...
+; table keyboard to ASCII
+;
+; This table is used to prepare keyboard
+; input for sending over the serial
+; line.
+;
+; ascii = $82 means HELP key
+; ascii = $81 means ignore key
+; ascii = $80 means do something
+;             complicated (command key)
```
Comment (oliverschmidt) [inline drivers/atrvt100.s]: Is there a special reason why you moved away from using $ff, $fe, ... downwards for special purposes? If not I'd appreciate if you would stick to the original approach used on the C64 and Apple II.

File: drivers/atrvt100.s [hunk 2]
```diff
@@ -1,6 +1,1709 @@
...
+; *************************************
+
+; -------------------------------------
+; table line drawing to ScreenCode
+;
+; This tabel is used to convert incoming
+; line drawing chars.
+; -------------------------------------
+
+ltsc;_0  _1  _2  _3  _4  _5  _6  _7  _8  _9  _a  _b  _c  _d  _e  _f
```
Comment (oliverschmidt) [inline drivers/atrvt100.s]: Do I understand it correctly that this whole table isn't actually adjusted to the Atari yet?

File: drivers/atrvt100.s (hunk 1 above)
Comment (groessler) [inline drivers/atrvt100.s]: I wasn't looking that much at the C64 version. I didn't notice that there's some kind of "convention". Looking at it now, I think most of what is used now for $80 shoult rather be $ff (send string).

File: drivers/atrvt100.s (hunk 2 above)
Comment (groessler) [inline drivers/atrvt100.s]: Adjusted, but incompletely. A better (and shorter) table will come with the font change.

File: drivers/atrvt100.s (hunk 1 above)
Comment (groessler) [inline drivers/atrvt100.s]: I've updated the PR....

**Comment by oliverschmidt** [comment]
Thanks for your contribution :-)

**Pull request merged**